src/app/pages/class-details/class-details.container/class-details.container.component.ts

```diff
@@ -0,0 +1,155 @@
+import { Component, OnInit } from '@angular/core';
+import { ActivatedRoute } from '@angular/router';
+import { TimeSlot } from '../../../models/timeslot.model';
+import { ClassService } from '../../class/services/class.graphql.service';
+import { TimeSlotIndexes } from '../../../components/schedule/schedule.component';
+import { MatDialog } from '@angular/material';
+import { ScheduleDialogComponent } from '../../../components/schedule/schedule-dialog/schedule.dialog';
+import { ScheduleDialogData } from '../../../components/schedule/schedule-dialog/schedule-dialog-data.model';
+import { Class } from '../../../models/class.model';
+import { ClassDetailsEventParams } from '../class-details.view/class-details.view.component';
+import { ScheduleService } from '../../../services/schedule/schedule.service';
+
+@Component({
+  selector: 'app-class-details-container',
+  template: `<app-class-details-view
```

Comment:
Don't we want to use templateUrl?

Reply:
seems redundant to create a another html file and request it for just a few lines of html